pghoard/restore.py

```diff
@@ -396,9 +400,17 @@ def _get_basebackup(self, pgdata, basebackup, site,
             # Object is a raw (encrypted, compressed) basebackup
             basebackup_data_files = [[basebackup, -1]]
 
+        if tablespace_base_dir and os.path.exists(tablespace_base_dir):
+            # we just care that the dir exists, but we're OK if there are other objects there
+            raise RestoreError("Tablespace base directory does not {!r} exist, aborting."
```

Comment:
I think you should remove the `not` from here.

Reply:
The message still should be modified a bit -> `directory {!r} does not exist`